src/components/v5/frame/ColonyHome/partials/ChartCustomTooltip/ChartCustomTooltip.tsx

```diff
@@ -1,13 +1,35 @@
 import { useTooltipState } from '@nivo/tooltip';
 import { AnimatePresence, motion } from 'framer-motion';
-import React, { type PropsWithChildren, type FC } from 'react';
+import React, { type PropsWithChildren, type FC, useEffect } from 'react';
 
-interface ChartCustomTooltipProps extends PropsWithChildren {}
+interface ChartCustomTooltipProps extends PropsWithChildren {
+  onVisible?: (isVisible?: boolean) => void;
+}
 
 export const ChartCustomTooltip: FC<ChartCustomTooltipProps> = ({
   children,
+  onVisible,
 }) => {
   const { isVisible } = useTooltipState();
+
+  useEffect(() => {
```

Comment:
Would it work to use the `onActiveIdChange` for the pie chart? https://nivo.rocks/pie/ supports this and programatically setting the `activeId` which we could then also add to clicking/hovering the legend item.
I think we can keep the context as it is, but just use the API instead of the tooltip for setting the activeId into the context

Reply:
Will convert it to draft until we explore this change 🙌